src/Trixi.jl

```diff
@@ -247,6 +247,11 @@ export ViscousFormulationBassiRebay1, ViscousFormulationLocalDG
 # Visualization-related exports
 export PlotData1D, PlotData2D, ScalarPlotData2D, getmesh, adapt_to_mesh_level!, adapt_to_mesh_level
 
+# Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
+@static if VERSION >= v"1.10.0-DEV.1288" || VERSION >= v"1.9.1"
```

Comment:
```suggestion
# We do not check `isdefined(Base, :get_extension)` since Julia v1.9.0
# does not load package extensions when their dependency is loaded from
# the main environment
@static if VERSION >= v"1.9.1"
```

The other branch does not help since
```julia
julia> v"1.10.0-DEV.1" >= v"1.9.1"
true
```

Reply:
Right, thanks!